nautobot/extras/api/serializers.py

```diff
@@ -293,6 +294,17 @@ def get_owner(self, obj):
         return return_nested_serializer_data_based_on_depth(self, depth, obj, obj.owner, "owner")
 
 
+#
+# External integrations
+#
+
+
+class ExternalIntegrationSerializer(NautobotModelSerializer):
+    class Meta:
+        model = ExternalIntegration
+        fields = "__all__"
```

Comment:
Question: does this expose any sensitive information from the API side?

Reply:
It shouldn't unless you're putting credentials in the URL which you shouldn't be doing.